apps/hatchet-worker-response-processor/.env.example

```
@@ -0,0 +1,6 @@
+HATCHET_CLIENT_TOKEN=__HATCHET_CLIENT_TOKEN__
+HATCHET_CLIENT_TLS_STRATEGY=none
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HATCHET_CLIENT_TLS_STRATEGY key should go before the HATCHET_CLIENT_TOKEN key

(UnorderedKey)